PluginBuilder/Data/Scripts/08.MultiVersionReturn.sql

```diff
@@ -1,12 +1,46 @@
+ALTER TABLE versions ADD COLUMN download_stat BIGINT NOT NULL DEFAULT 0;
+
+ALTER TABLE evts
+ADD COLUMN plugin_slug TEXT,
+ADD COLUMN build_id BIGINT,
+ALTER COLUMN type TYPE VARCHAR(16),
+ADD COLUMN id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+ADD COLUMN ip VARCHAR(45),
+ADD CONSTRAINT unique_event UNIQUE (ip, plugin_slug, type);
+
+CREATE INDEX idx_evts_plugin_slug ON evts(plugin_slug);
+CREATE INDEX idx_evts_type ON evts(type);
+CREATE INDEX idx_evts_build_id ON evts(build_id);
```

Comment:
why `idx_evts_build_id` and `idx_evts_type` are necessary?

Reply:
Might not be core for now, but should incase we would need to run queries/filtering on the event type it would come a good to have